frontend/packages/dev-console/integration-tests/support/pages/helm/controls.ts

```diff
@@ -0,0 +1,18 @@
+export const controls = {
+  dropdown: {
+    switchTo: (newOption: string) => {
+      cy.byLegacyTestID('dropdown-button')
+        .click()
+        .get('.pf-c-dropdown__menu')
+        .contains(newOption)
+        .click();
+    },
+    verifyVisibility: () => {
+      cy.byLegacyTestID('dropdown-button').should('be.visible');
+      cy.byLegacyTestID('dropdown-button')
+        .click()
+        .get('.pf-c-dropdown__menu')
+        .should('be.visible');
+    },
+  },
+};
```

Comment:
Recommended to move it to commands folder

Reply:
sure